src/main/java/uk/gov/dvsa/mot/framework/WebDriverWrapper.java

```diff
@@ -1143,12 +1143,12 @@ public void selectOptionInFieldById(String optionText, String id) {
      */
     public void checkCurrentPageTitle(String expected) {
         String actual = webDriver.getTitle();
-        logger.debug("Checking current page title for '{}' contains '{}'", actual, expected);
-        if (!actual.contains(expected)) {
+        logger.debug("Checking current page title for '{}' contains '{}'", actual, expandDataKeys(expected));
+        if (!actual.contains(expandDataKeys(expected))) {
```

Comment:
Do you actually need / use this?

Reply:
This was going to be used but in the end I didn't. I will remove this change